packages/client-adapters/src/adapters/lexicons/adapter.js

```diff
@@ -485,7 +485,7 @@ class AlpheiosLexiconsAdapter extends BaseAdapter {
           lastAlt.push(strippedAll[0])
         }
       }
-      if (lastAlt.length > 0) {
+      if (data && lastAlt.length > 0) {

```

Comment:
Added for the cases when data is uploaded from remote source too long, and user has time to request a new query.
It is a bug fix for showing error when data is not ready (it is not for the min task of the PR)